<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOSSelectDialog</class>
 <widget class="QDialog" name="IOSSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IOSSelectDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#IOSSelectDialog
{
	border-image: url(:/Live/Resources/HandTour_Game.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget
{
	background-color: rgb(33, 33, 33);
}</string>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>496</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget_2
{
	
	background-color: rgb(43, 43, 43);
}
</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>9</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 126, 24);</string>
       </property>
       <property name="text">
        <string>IOS直播</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>8</y>
         <width>15</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/SeedUI/Resources/SeedUI/SeedUI_iOS.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Close_pushButton">
       <property name="geometry">
        <rect>
         <x>476</x>
         <y>10</y>
         <width>10</width>
         <height>10</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Close_pushButton{
	
	border-image: url(:/Close/Resources/Close_Normal.png);
}
QPushButton#Close_pushButton:hover{
	
	border-image: url(:/Close/Resources/Close_Hover.png);
}
QPushButton#Close_pushButton:disabled{
	
	border-image: url(:/Close/Resources/Close_Disabled.png);
}
QPushButton#Close_pushButton:checked{
	
	border-image: url(:/Close/Resources/Close_checked.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Download_pushButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>270</y>
        <width>83</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton#Download_pushButton
{
	border-image: url(:/SeedUI/Resources/SeedUI/DownloadButton.png);
	font: 75 12px &quot;微软雅黑&quot;;
	color: rgb(240, 240, 240);
}

QPushButton#Download_pushButton:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/DownloadButton_H.png);
}

QPushButton#Download_pushButton:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/DownloadButton_C.png);
}
</string>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>391</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 85);
background-color: rgb(33, 33, 33);
border:px groove gray;
</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;1、运行电脑上的itools苹果录制大师Airplayer。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;2、在手机底部向上滑动打开Airplay选择itools镜像。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;3、确认手机Airplay开始,点击下方添加即可开始录制IOS。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>31</y>
        <width>93</width>
        <height>274</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget_3
{
	
	background-color: rgb(43, 43, 43);
}
</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color: rgb(43, 43, 43);
	font: 75 14px &quot;微软雅黑&quot;;
	color: rgb(254, 161, 24);
	border:px groove gray;
}

QPushButton:hover
{
	background-color: rgb(70, 70, 70);
}

QPushButton:checked
{
	background-color: rgb(33, 33, 33);
}

QPushButton:disabled
{
	background-color: rgb(33, 33, 33);
}

</string>
         </property>
         <property name="text">
          <string>Airplay</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>130</y>
        <width>381</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/IOS/Resources/IOS.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="IOS_pushButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#IOS_pushButton
{
	border-image: url(:/SeedUI/Resources/SeedUI/DownloadButton.png);
	font: 75 12px &quot;微软雅黑&quot;;
	color: rgb(240, 240, 240);
}

QPushButton#IOS_pushButton:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/DownloadButton_H.png);
}

QPushButton#IOS_pushButton:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/DownloadButton_C.png);
}
</string>
      </property>
      <property name="text">
       <string>下载Airplay</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>44</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(254, 161, 24);
font: 70 14px &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>提示：请确保手机和电脑连接的为同意wifi</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rdzbdemo.qrc"/>
 </resources>
 <connections/>
</ui>
